content/en/Getting started/glossary.md

```diff
@@ -263,6 +269,14 @@ wired, analog, or digital.
 Operations Security, commonly known as OpSec, identifies critical information, and if/how it
 may be used by opponents or enemies. OpSec measures can reduce security risks.
 
+## Page
```

Comment:
I'd really like a different term, something that doesn't remind me of reading a book. Maybe:

```suggestion
## Web Page Types
```

Reply:
@mjang-cobalt "Page" is the term we're using in the UI for Web assets in the upcoming release. Currently, it's "Dynamic Page." I would rename it to Web Page and give a short definition such as "A hypertext document on the World Wide Web."